src/apis/faq/index.ts

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+import {standardHeaders} from 'utils/requests';
+
+export async function sendFeedback(name: string, email: string, response: string) {
+  return axios.post(
+    `https://v1.nocodeapi.com/jamessspanggg/google_sheets/XKBFrUzDbyXOOFNR?tabId=Sheet1&api_key=${process.env.REACT_APP_GOOGLE_SHEET_FAQ_API_KEY}`,
```

Comment:
@jamessspanggg what is this endpoint? Is it a test endpoint, or the final one that we will be using in PROD?

Reply:
this is a test endpoint that we will be using to integrate the contact us submission to google sheets. I think we should create one using our team's email and use that. I stumbled upon this solution via googling, not sure if this is the best solution, also not too sure if we want to store this in google docs or in our BE.

https://nocodeapi.com/integrate-google-sheets-with-your-feedback-form